notebooks/s3d-mil-nce-text-to-video-retrieval/s3d-mil-nce-text-to-video-retrieval.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    hub_handle = "https://tfhub.dev/deepmind/mil-nce/s3d/1"
</pre><p>tf hub was migrated into kaggle as source for storing models. This link will redirect on it during downloading. In some setup and slow connection it may lead to long downloading and even timeout issues. Could you please update link using kaggle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>replaced</p>